# let PIL finds the EB deps
# By Ward Poelmans <ward.poelmans@ugent.be>
diff -ur Imaging-1.1.7.orig/setup.py Imaging-1.1.7/setup.py
--- Imaging-1.1.7.orig/setup.py	2009-11-15 17:06:10.000000000 +0100
+++ Imaging-1.1.7/setup.py	2016-01-07 12:29:07.708559953 +0100
@@ -14,6 +14,8 @@
     pass
 
 def libinclude(root):
+    if not root:
+        return None
     # map root to (root/lib, root/include)
     return os.path.join(root, "lib"), os.path.join(root, "include")
 
@@ -33,11 +35,11 @@
 #
 # TIFF_ROOT = libinclude("/opt/tiff")
 
-TCL_ROOT = None
-JPEG_ROOT = None
-ZLIB_ROOT = None
-TIFF_ROOT = None
-FREETYPE_ROOT = None
+TCL_ROOT = libinclude(os.getenv("EBROOTTCL"))
+JPEG_ROOT = libinclude(os.getenv("EBROOTLIBJPEGMINTURBO"))
+ZLIB_ROOT = libinclude(os.getenv("EBROOTZLIB"))
+TIFF_ROOT = libinclude(os.getenv("EBROOTLIBTIFF"))
+FREETYPE_ROOT = libinclude(os.getenv("EBROOTFREETYPE"))
 LCMS_ROOT = None
 
 # FIXME: add mechanism to explicitly *disable* the use of a library
